funcs/cel_gen_exports.go

```diff
@@ -0,0 +1,264 @@
+	dataToJSONGen,
+	dataToJSONPrettyGen,
+	dataToYAMLGen,
+	dataToTOMLGen,
+	envGetenvGen,
+	envExpandEnvGen,
+	fileReadGen,
+	fileStatGen,
+	fileExistsGen,
+	fileIsDirGen,
+	fileReadDirGen,
+	fileWalkGen,
+	fileWriteGen,
+	filepathBaseGen,
+	filepathCleanGen,
+	filepathDirGen,
+	filepathExtGen,
+	filepathFromSlashGen,
+	filepathIsAbsGen,
+	filepathJoinGen,
+	filepathMatchGen,
+	filepathRelGen,
+	filepathSplitGen,
+	filepathToSlashGen,
+	filepathVolumeNameGen,
+	gcpMetaGen,
+	mathIsIntGen,
+	mathIsFloatGen,
+	mathcontainsFloatGen,
+	mathIsNumGen,
+	mathAbsGen,
+	mathAddGen,
+	mathMulGen,
+	mathSubGen,
+	mathDivGen,
+	mathRemGen,
+	mathPowGen,
+	mathSeqGen,
+	mathMaxGen,
+	mathMinGen,
+	mathCeilGen,
+	mathFloorGen,
+	mathRoundGen,
+	netLookupIPGen,
+	netLookupIPsGen,
+	netLookupCNAMEGen,
+	netLookupSRVGen,
+	netLookupSRVsGen,
+	netLookupTXTGen,
+	netParseIPGen,
+	netParseIPPrefixGen,
+	netParseIPRangeGen,
+	netStdParseIPGen,
+	netstdParseCIDRGen,
+	netStdParseCIDRGen,
+	netCidrHostGen,
+	netCidrNetmaskGen,
+	netCidrSubnetsGen,
+	netCidrSubnetSizesGen,
+	pathBaseGen,
+	pathCleanGen,
+	pathDirGen,
+	pathExtGen,
+	pathIsAbsGen,
+	pathJoinGen,
+	pathMatchGen,
+	pathSplitGen,
+	randomASCIIGen,
+	randomAlphaGen,
+	randomAlphaNumGen,
+	randomStringGen,
+	randomItemGen,
+	randomNumberGen,
+	randomFloatGen,
+	regexpFindGen,
+	regexpFindAllGen,
+	regexpMatchGen,
+	regexpQuoteMetaGen,
+	regexpReplaceGen,
+	regexpReplaceLiteralGen,
+	regexpSplitGen,
+	sockaddrGetAllInterfacesGen,
+	sockaddrGetDefaultInterfacesGen,
+	sockaddrGetPrivateInterfacesGen,
+	sockaddrGetPublicInterfacesGen,
+	sockaddrSortGen,
+	sockaddrExcludeGen,
+	sockaddrIncludeGen,
+	sockaddrAttrGen,
+	sockaddrJoinGen,
+	sockaddrLimitGen,
+	sockaddrOffsetGen,
+	sockaddrUniqueGen,
+	sockaddrMathGen,
+	sockaddrGetPrivateIPGen,
+	sockaddrGetPrivateIPsGen,
+	sockaddrGetPublicIPGen,
+	sockaddrGetPublicIPsGen,
+	sockaddrGetInterfaceIPGen,
+	sockaddrGetInterfaceIPsGen,
+	stringsAbbrevGen,
+	stringsReplaceAllGen,
+	stringsContainsGen,
+	stringsHasPrefixGen,
+	stringsHasSuffixGen,
+	stringsRepeatGen,
+	stringsSortGen,
+	stringsSplitGen,
+	stringsSplitNGen,
+	stringsTrimGen,
+	stringsTrimPrefixGen,
+	stringsTrimSuffixGen,
+	stringsTitleGen,
+	stringsToUpperGen,
+	stringsToLowerGen,
+	stringsTrimSpaceGen,
+	stringsTruncGen,
+	stringsIndentGen,
+	stringsSlugGen,
+	stringsQuoteGen,
+	stringsShellQuoteGen,
+	stringsSquoteGen,
+	stringsSnakeCaseGen,
+	stringsCamelCaseGen,
+	stringsKebabCaseGen,
+	stringsWordWrapGen,
+	stringsRuneCountGen,
+	testAssertGen,
+	testFailGen,
+	testRequiredGen,
+	testTernaryGen,
+	testKindGen,
+	testIsKindGen,
+	timeZoneNameGen,
+	timeZoneOffsetGen,
+	timeParseGen,
+	timeParseLocalGen,
+	timeParseInLocationGen,
+	timeNowGen,
+	timeUnixGen,
+	timeNanosecondGen,
+	timeMicrosecondGen,
+	timeMillisecondGen,
+	timeSecondGen,
+	timeMinuteGen,
+	timeHourGen,
+	timeParseDurationGen,
+	timeSinceGen,
+	timeUntilGen,
+	uuidV1Gen,
+	uuidV4Gen,
+	uuidNilGen,
+	uuidIsValidGen,
+	uuidParseGen,
+}
```

Comment:
```suggestion
var CelEnvOption = []cel.EnvOption{	
	collDictGen,
	collKeysGen,
	collValuesGen,
	collAppendGen,
	collPrependGen,
	// use aliases instead of coll.Uniq
	collUniqGen,
	collReverseGen,
	collMergeGen,
	collSortGen,
	collJSONPathGen,
	collFlattenGen,
	collPickGen,
	collOmitGen,
	convBoolGen,
	convToBoolGen,
	convToBoolsGen,
	convSliceGen,
	convParseIntGen,
	convParseFloatGen,
	convParseUintGen,
	convAtoiGen,
	convURLGen,
	convToInt64Gen,
	convToIntGen,
	convToInt64sGen,
	convToIntsGen,
	convToFloat64Gen,
	convToFloat64sGen,
	convToStringGen,
	convToStringsGen,
	convDefaultGen,
	convDictGen,
	cryptoSHA1Gen,
	cryptoSHA256Gen,
	cryptoSHA384Gen,
	// use the aliases  e.g. json or toJSON
	dataJSONGen,
	dataJSONArrayGen,
	dataYAMLGen,
	dataYAMLArrayGen,
	dataTOMLGen,
	dataCSVGen,
	dataCSVByRowGen,
	dataCSVByColumnGen,
	dataToCSVGen,
	dataToJSONGen,
	dataToJSONPrettyGen,
	dataToYAMLGen,
	dataToTOMLGen,
	filepathBaseGen,
	filepathCleanGen,
	filepathDirGen,
	filepathExtGen,
	filepathFromSlashGen,
	filepathIsAbsGen,
	filepathJoinGen,
	filepathMatchGen,
	filepathRelGen,
	filepathSplitGen,
	filepathToSlashGen,
	filepathVolumeNameGen,
	mathIsIntGen,
	mathIsFloatGen,
	mathcontainsFloatGen,
	mathIsNumGen,
	mathAbsGen,
	mathAddGen,
	mathMulGen,
	mathSubGen,
	mathDivGen,
	mathRemGen,
	mathPowGen,
	mathSeqGen,
	mathMaxGen,
	mathMinGen,
	mathCeilGen,
	mathFloorGen,
	mathRoundGen,
	netParseIPGen,
	netParseIPPrefixGen,
	netParseIPRangeGen,
	netStdParseIPGen,
	netstdParseCIDRGen,
	netStdParseCIDRGen,
	netCidrHostGen,
	netCidrNetmaskGen,
	netCidrSubnetsGen,
	netCidrSubnetSizesGen,
	pathBaseGen,
	pathCleanGen,
	pathDirGen,
	pathExtGen,
	pathIsAbsGen,
	pathJoinGen,
	pathMatchGen,
	pathSplitGen,
	randomASCIIGen,
	randomAlphaGen,
	randomAlphaNumGen,
	randomStringGen,
	randomItemGen,
	randomNumberGen,
	randomFloatGen,
	regexpFindGen,
	regexpFindAllGen,
	regexpMatchGen,
	regexpReplaceGen,
	regexpSplitGen,
	stringsAbbrevGen,
	stringsReplaceAllGen,
	stringsContainsGen,
	stringsRepeatGen,
	stringsSortGen,
	stringsTrimPrefixGen,
	stringsTrimSuffixGen,
	stringsTitleGen,
	stringsToUpperGen,
	stringsToLowerGen,
	stringsTruncGen,
	stringsIndentGen,
	stringsSlugGen,
	stringsQuoteGen,
	stringsShellQuoteGen,
	stringsSquoteGen,
	stringsSnakeCaseGen,
	stringsCamelCaseGen,
	stringsKebabCaseGen,
	stringsWordWrapGen,
	stringsRuneCountGen,
	timeParseGen,
	timeParseLocalGen,
	timeParseInLocationGen,
	timeUnixGen,
	timeNanosecondGen,
	timeMicrosecondGen,
	timeMillisecondGen,
	timeSecondGen,
	timeMinuteGen,
	timeHourGen,
	timeParseDurationGen,
	timeSinceGen,
	timeUntilGen,
	uuidV1Gen,
	uuidV4Gen,
	uuidNilGen,
	uuidIsValidGen,
	uuidParseGen,
}
```